render/mapping.go

```diff
@@ -441,8 +441,10 @@ func MapProcess2Container(n RenderableNode, _ report.Networks) RenderableNodes {
 		id = MakeContainerID(containerID)
 		node = NewDerivedNode(id, n)
 	} else {
+		nCopy := n.Copy()
+		nCopy.Node = nCopy.Node.WithID("").WithTopology("") // Wipe the ID so it cannot be rendered.
```

Comment:
Does this mean we won't show any information about the process? Not even the process itself? Why?